node_monitor/node_monitor_helpers/messages.py

```diff
@@ -125,5 +128,5 @@ def _render_frac(numerator: int, denominator: int) -> str:
         f"Thanks for reviewing today's report. We'll be back tomorrow!\n"
         f"Node Monitor by Aviate Labs.\n"
         f"Report generated: {datetime.utcnow().isoformat()} UTC\n"
-        f"Help us serve you better! Provide your feedback!\n")
+        f"Help us serve you better! Provide your feedback here: {feedback_form} \n")
```

Comment:
Why not just 
```py
import node_monitor.load_config as c
```

At the top of this file? That way we don't have to change the function interface?

Reply:
Because if we do it this way, then we cannot put a fake form when running tests. I thought it was important to do this especially in the case when we send mails to the public email inbox.